landing-pages/site/content/en/blog/airflow-2.7.0/index.md

```diff
@@ -0,0 +1,84 @@
+
+## Enable deferrable mode for all deferable tasks with 1 config setting
+
+Airflow 2.7.0 comes with a new config option, `default_deferrable`, which allows admins to enable deferrable mode for all deferrable tasks without requiring any DAG modifications. Simply set it in your config and enjoy async tasks!
+
+## OpenLineage provider
+
+[OpenLineage](https://openlineage.io/) provides a spec standardizing operational lineage collection and distribution across the data ecosystem that projects – open source or proprietary – implement.
+
+With 2.7.0, OpenLineage changes from a plugin implementation maintained in the OpenLineage project to a built-in feature of Airflow. As a plugin, OpenLineage depended on Airflow and operators’ internals, making it brittle. Built-in OpenLineage support in Airflow makes publishing operational lineage through the OpenLineage ecosystem easier and more reliable. It has been implemented by moving the [openlineage-airflow](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow) package from the OpenLineage project to an `apache-airflow-providers-openlineage` provider in the base Airflow Docker image, where it can be easily enabled by configuration. Also, lineage extraction logic that was included in [Extractors](https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow/openlineage/airflow/extractors) in that package has been moved into each corresponding provider package along with unit tests, eliminating the need for Extractors in most cases. For this purpose, a new optional API for Operators (`get_openlineage_facets_on_{start(), complete(ti), failure(ti)}`, documented [here](https://openlineage.io/docs/integrations/airflow/default-extractors)) can be used. Having the extraction logic in each provider ensures the stability of the lineage contract in each operator and makes adding lineage coverage to custom operators easier.
+
+## Some executors moved into providers
+
+Some of the executors that were shipped in core Airflow have moved into their respective providers for Airflow 2.7.0.
+
+The following providers have been moved and require certain minimum providers versions:
+
+* In order to use Celery executors, install the celery provider version 3.3.0+
+* In order to use the Kubernetes executor, install the kubernetes provider version 7.4.0+
+* In order to use the Dask executor, install any version of the daskexecutor provider
+
+If you use the official docker images, all of these providers come preinstalled.
+
+## Additional new features
+
+Here are just a few interesting new features, since there are too many to list in full:
+
+* Pools can now consider tasks in the deferred state as running (#32709)
+* chain_linear, like chain but allowing sequential tasks (#31927)
+* Grid view now supports keyboard shortcuts! (#30950)
+* Mark task groups as success or failed (#30478)
+* fail_stop, allowing all remaining and running tasks to be failed on the first failure in a DAG (#29406)
+
```

Comment:
I think there are few more things worth mentioning here:

* this is the first release of airflow that dropped support for (end of life) Python 3.7 and mentioning that it follows the community and allows to use new features available in 3.8.


I was wondering if we should mention security model and disabling test connection by default. I think it could be acompanied by the strenghtening of focus of Airflow on security and having it displayed here as deliberate action might help to fend-off people who will complain.

While both are kind of "take away" things, they are also explaining that we are looking forward and improving the "infrastructure" of Airflow - and both have quite an impact on airflow "overall".

Reply:
No objections to either of those - in particular I like the security model/connections one. Feel free to ghost write them for me, or I'll do it first thing when I wake up tomorrow.

Reply:
👻 ✍️ 👍 then:

* Airflow 2.7.0 is the first release that drops support for end-of-life Python 3.7. This allows Airflow users and maintainer to make use of features and improvements in Python 3.8, and newer versions of our dependencies (#30963)

* Airflow 2.7.0 is the release that focuses on security. The airflow security team, working together with security researchers identified a number of areas that required strenghtening of security - resulting, among others, in improved description of [Airflow security model](https://airflow.apache.org/docs/apache-airflow/stable/security/security_model/), better explanation of our [security policy](https://github.com/apache/airflow/security/policy) and disabling certain, potentially dangerous features by default - like for example Test Connection (#32052)

Reply:
Added both cc @potiuk